### ISSUE TITLE
fastboot: error: cannot load 'vbmeta.img': No such file or directory

### ISSUE THREAD

**Issue by semorebutts123** [fu8765/Lenovo-Tab-M10-Plug-Gen-3-FlashiGSI-roms#1]
Hey mate I've been trying to follow your instructions but I keep getting this error on step 4 when I try the fastboot flash --disable-verity --disable-verification vbmeta vbmeta.img command

fastboot: error: cannot load 'vbmeta.img': No such file or directory

**Comment by ComradeWave** [comment]
Do you have the vbmeta.img in the same directory of your command?

**Comment by fu8765** [comment]
Yes, i believe i extracted the vbmeta from the stock rom. I should be receiving my new tablet soon and need to reflash, I'll update instructions once i do that.

But if i remember correctly i got the stock rom from the Lenovo rescue assistant, let it download the image and look for the file in a temp folder of some sort before you click on flashing it. Copy it somewhere else then cancel RSA.

Then you can extract it

**Comment by justbendev** [comment]
**To flash vbmeta you need to reboot in the bootloader using** 
```bash
fastboot reboot bootloader
```
